documents/pdf.js

```diff
@@ -0,0 +1,89 @@
+module.exports = (activities, users) => {
```

Comment:
you passed users[0] to the second parameter in generatePDF.js, so you get here a user: why did you name it like an array of user?